src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,46 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	private static final int FULL_STACK = 11;
```

Comment:
This constant is related to STACK_CAPACITY. Changing only one leads to an error (inconsistency).